<launch>
  <node name="urimits" pkg="urimits" type="urimits_exec" output="screen" ns="/AS/P">
    <rosparam command="load" file="$(find urimits)/params/urimits.yml" />
    <param name="compute_short_tls" value="true"/>
    <param name="compute_loop" value="false"/>
  </node>
</launch>